homeassistant/components/camera/amcrest.py

```diff
@@ -6,23 +6,31 @@
 """
 import logging
 
+import asyncio
+import aiohttp
+from aiohttp import web
+from aiohttp.web_exceptions import HTTPGatewayTimeout
+import async_timeout
 import voluptuous as vol
 
 import homeassistant.loader as loader
 from homeassistant.components.camera import (Camera, PLATFORM_SCHEMA)
 from homeassistant.const import (
     CONF_HOST, CONF_NAME, CONF_USERNAME, CONF_PASSWORD, CONF_PORT)
 from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
 
 REQUIREMENTS = ['amcrest==1.0.0']
```

Comment:
You can bump this version to 1.0.1 and we can align and publish it prior getting this merged to HA